VogtPayroll5/FileSystem.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+
+namespace VogtPayroll5
+{
+    class FileSystem
```

Comment:
Oooo! good to have this separated into its own code. Not too big of a deal, but one level further is to make this a generic WriteToFile(string text, strng fileName)

then, you would use a StringBuilder() in the previous class to build the text. Note that you use `\n` for newline when building a string, then you could just do a st.Write(text);